deploy_slave.sh

```diff
@@ -64,7 +64,7 @@ apt -y install vim ssh-import-id tree byobu htop pkg-config cmake time \
     libtiff5-dev libx11-6 libx11-dev xfonts-base xfonts-100dpi xfonts-75dpi \
     xfonts-cyrillic fluid-soundfont-gm libsystemd-dev libusb-1.0-0-dev \
     libudev-dev libopus-dev libvpx-dev libc-bin libavdevice-dev \
-    libavfilter-dev libavutil-dev lsb-release
+    libadios-dev libavfilter-dev libavutil-dev lsb-release
```

Comment:
hm, is this even in raspbian?
It's on all official debian architectures: https://packages.debian.org/buster/libadios-dev

raspbian arch? http://archive.raspberrypi.org/debian/pool/main/

Reply:
Hm, according to this index it should be there, somewhere: https://archive.raspbian.org/raspbian/dists/buster/main/binary-armhf/